lib/publish.js

```diff
@@ -1,43 +1,66 @@
 const execa = require('execa')
+const semver = require('semver')
```

Comment:
Please add semver to `dependencies` in `package.json` and `package-lock.json` :pray:

Reply:
Sorry for that, just added!